ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -4,8 +4,8 @@
   apt: name=ecryptfs-utils state=present
 
 - name: Check for already mounted drive
-  shell: cat /etc/fstab
-  register: fstab_contents
+  shell: cat /etc/mtab
+  register: mtab_contents
```

Comment:
Ok, so you're saying this was always wrong, and it should always have been mtab.